corehq/apps/app_manager/static/app_manager/js/preview_app.js

```diff
@@ -102,6 +102,9 @@ hqDefine('app_manager/js/preview_app.js', function() {
         } else {
             _private.phoneView(true);
         }
+        setTimeout(function () {
+            $(window).trigger(module.EVENTS.RESIZE);
+        }, 1001);
```

Comment:
nice may want a comment in the future as i'm assuming this is 1001 inorder to be after timers that are 1000